### ISSUE TITLE
Main PE window must remain open for PE to work

### ISSUE THREAD

**Issue by addeps3** [boomshop/pulseffectstray#3]
I was hoping that this tray would enable me to minimize PE to the tray, but that is not the intention, I gather.

**Comment by boomshop** [comment]
No, intention was to have a tiny control for PE, toggling bypass and applying all available presets. Aside from killing PE completely.

**Comment by boomshop** [comment]
For controlling the window it would be much better to have it implemented in PE itself, which isn't planned afaik.
